Package.swift

```diff
@@ -16,15 +16,21 @@ let package = Package(
     ],
     dependencies: [
         // 💧 A server-side Swift web framework.
-        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0")
+        .package(url: "https://github.com/vapor/vapor.git", from: "4.0.0"),
```

Comment:
Need resolve this error error: The package product 'Vapor' requires minimum platform version 13.0 for the iOS platform, but this target supports 11.0 (in target 'AppHealthChecks' from project 'fs-app-health-checks')